practice/src/javascripts/templates/product.js

```diff
@@ -0,0 +1,78 @@
+import iconAction from '../../assets/images/icons/icons.svg';
+
+/**
+ * View template for rendering product cards
+ */
+export default class ProductListView {
+  /**
+   * Clear the main container
+   */
+  static clearMainContainer() {
```

Comment:
Please help me explain more why we need to use static here.

Reply:
Because I want emphasizing that they are independent of any specific instance of the `ProductTemplate` class. They can be called directly on the class itself without needing to instantiate it. But I can remove it and create an instance of `ProductTemplate` class.